FirebaseAuth/Sources/Swift/Auth/Auth.swift

```diff
@@ -93,7 +93,7 @@ extension Auth: AuthInterop {
           strongSelf.autoRefreshTokens = true
           strongSelf.scheduleAutoTokenRefresh()
 
-          #if os(iOS) || os(tvOS) // TODO: Is a similar mechanism needed on macOS?
+          #if os(iOS) || os(tvOS) // TODO(from ObjC): Is a similar mechanism needed on macOS?
```

Comment:
nit: `TODO(ObjC)` to match the ones below

Reply:
fixed